recipes/portion/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "portion" %}
+{% set version = "2.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 5b9200a360c576b342f653502cf4dd04e327a0f385e97265a70cc7fa8ad0d47f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - portion
+
+about:
+  home: "https://github.com/AlexandreDecan/portion"
+  license: "LGPL-3.0-only"
+  license_family: LGPL
+  license_file: LICENSE.txt
+  summary: "Python data structure and operations for intervals"
+
+extra:
+  recipe-maintainers:
+    - AlexandreDecan
```

Comment:
@AlexandreDeca
Could you please confirm that you also want to maintain this recipe?

Reply:
I think the other maintainer should be: @AlexandreDecan (typo in the name)

Reply:
Sorry! exactly that :smile:

Reply:
I confirm ;-)